Kudu.Contracts/Deployment/DeploymentInfoBase.cs

```diff
@@ -68,5 +69,9 @@ public bool IsValid()
         // won't update until after a process restart. Therefore, we copy the needed
         // files into a separate folders and run sync triggers from there.
         public string SyncFunctionsTriggersPath { get; set; } = null;
+
+        public string FileName { get; set; }
```

Comment:
>FileName [](start = 22, length = 8)

Move this closer to "TargetPath". Also, add comments in both so that it is clear what the difference is and why we need both.